draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -395,8 +395,28 @@ discussed in Sections 6, 11.1, and 12 of {{?RFC3649}}.
 
 ### Real-Time Protocols
 
-(TODO: Clarify that real time congestion controls are included, with
-allowances for the poor documentation / open source availability of these)
+General-purpose protocols coexist in the internet with real-time congestion
+control protocols, which in general have finite throughput requirements and
+more strict latency bounds.
```

Comment:
These protocols are often not capacity seeking, and hence methods such as CWV might apply [RFC7661]?

Reply:
"finite throughput requirements" was my way of saying "non capacity seeking".  I don't think it's necessarily the role of this document to identify examples of specific congestion control proposals.

Reply:
Sure, we ought not to advocate methods... but I think  we could day "(i.e. does not seek to utilise all allowed capacity)" or something to indicate what we mean